### ISSUE TITLE
updating Tooltip to an element

### ISSUE THREAD

**Issue by lopey66** [twbs/bootstrap#37195]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

created a simple button to click on to update the tooltip title, original tooltip appears when hovering
-> on first the click the old tooltip still appears with a new tooltip appearing and original remains
-> on second click both tooltips remain visible 

I am only updating the tip with this function:

function fnUpdateBootStrapTitle(ele,new_tip) {
	var eleID=document.getElementById(ele);
	var tooltip = new bootstrap.Tooltip(eleID, {title:new_tip});
};

### Reduced test cases

https://lopeys.com/strava/bootstrap5tooltip2.html

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome

### What version of Bootstrap are you using?

5.2.1

**Comment by GeoSot** [comment]
I have made a [codepen](https://codepen.io/GeosSV/pen/gOzzWYV?editors=1011) to be able to study the whole thing. 
For sure, the thing you are trying to do, is not so much valid ;)

You are trying to re-set title, using a new instance of tooltip. You can use more valid ways to do it https://getbootstrap.com/docs/5.2/components/tooltips/#methods 😉 , or just dispose tooltip before re-create it


HOWEVER, you have discovered an insidious flaw here, which I will try to explain on another issue.

In short, you "cheated" creating duplicate instances of the same element. This is supposed to be forbidden by design



closing this comment as per above
